lib/govuk/diff/pages/html_diff/runner.rb

```diff
@@ -15,7 +12,7 @@ def self.assets_dir
           end
 
           def initialize(list_of_pages_uri:)
-            @list_of_pages_uri = list_of_pages_uri
+            @pages = list_of_pages_uri
```

Comment:
Actually on second thought - let's make this more consistent. Suggest we call this arg `paths` throughout.